src/core/lib/Magic.mjs

```diff
@@ -264,6 +246,114 @@ class Magic {
         return results;
     }
 
+    /**
+     * Checks the mime value of the data compared to the specified type.
+     *
+     * @param {ArrayBuffer} data
+     * @param {string} type
+     * @returns {boolean}
+     */
+    async checkMime (data, type) {
+        if (isType(type, data)) {
```

Comment:
If all this is doing is calling `isType`, it's probably not required. Just call `isType` where you're currently calling `checkMime`